src/components/features/Button/Button.tsx

```diff
@@ -0,0 +1,22 @@
+import { PropsWithChildren } from "react";
+import { ButtonAppearance, ButtonProperties } from "./Button.types";
+import classNames from "classnames";
+import Style from "./Button.module.scss";
+
+export const Button = ({
+  appearance = ButtonAppearance.Primary,
+  children,
+  ...otherProperties
+}: PropsWithChildren<ButtonProperties>) => {
+  return (
+    <button
+      {...otherProperties}
```

Comment:
Let’s create a Loader component and add it inside of the button with a loading state